vspace/src/main/webapp/WEB-INF/views/staff/exhibit/config.html

```diff
@@ -1,64 +1,74 @@
+		$(document).ready(function() {
+		    var offlineMessage = $('#offlineMessage');
+		    if ('[(${exhibition})]' != null && '[(${exhibition.mode})]' == 'OFFLINE') {
+		        offlineMessage.show();
+		    }
+		    else {
+		        offlineMessage.hide();
+		    }
+		});
+		
+		function modeChange(modeChosen) {
+		    var offlineMessage = $('#offlineMessage');
+		    if (modeChosen == 'OFFLINE') {
+		        offlineMessage.show();
+		    }
+		    else {
+		        offlineMessage.hide();
+		    }
+		}
+	</script>
+</head>
+
+<body>
+	<div layout:fragment="content" class="main-content">
+		<h2>Exhibition Configuration</h2>
+	
+		<div style="padding: 30px 0;">
+		    <form th:action="@{/staff/exhibit/config}" method="POST" name="f">
+		        <input type="hidden"  th:name="${_csrf.parameterName}" th:value="${_csrf.token}"/>
+		        <input type="hidden" th:value="${exhibition.id}" name="exhibitionParam"/>
+				<label for="space">Select the start space of the exhibition:</label>
+				<th:block th:if="${exhibition.startSpace}">
+		        <select class="form-control" name="spaceParam">
+		             <option th:id="${space.id}" th:each="space: ${spacesList}" th:value="${space.id}" 
+		              th:selected="${#strings.equals(space.id,exhibition.startSpace.id)}">
+		             	[[${space.name}]]
+		             </option>
+				</select>
+				</th:block>
+				<th:block th:if="${exhibition.startSpace == null}">
+		        	<select class="form-control" name="spaceParam">
+						<option th:id="choose">Choose...</option>
+		             	<option th:id="${space.id}" th:each="space: ${spacesList}" th:value="${space.id}">
+		             	[[${space.name}]]
+		             	</option>
+					</select>
+				</th:block>
+		        <div style="padding-top: 20px;">
+		            <label for="title">Exhibition Title:</label> 
+		            <input type="text" class="form-control" name="title" th:value="${exhibition.title}" />
+		        </div>
+		        <div style="padding-top: 20px;">
+		            <label for="title">Exhibition Mode:</label>
+		            <select class="form-control" name="exhibitMode" onChange="modeChange(exhibitMode.value)">
+		                <option th:each="mode: ${exhibitionModes}" th:id=${mode} th:value=${mode} th:selected="${#strings.equals(mode,exhibition.mode)}">[[${mode}]]</option>
+		            </select>
+		        </div>
+		        <div id="offlineMessage" style="padding-top: 20px;" th:style="${exhibition.mode != 'OFFLINE' ? 'display: none;' : '' }">
+		            <label for="title">Offline Message:</label> 
+		            <input type="text" class="form-control" name="customMessage"
+		                th:value="${exhibition.customMessage}" placeholder ="This exhibition is currently offline. Please check back later."/>
+		        </div>
+		        <p style="padding-top: 20px;">
+		            <input class="btn primary-btn" type="submit" value="Submit" />
+		        </p>	
+		    </form>
+		</div>
+	</div>
```

Comment:
did you actually change anything in this file? Please revert indentation changes so it's clear what was actually changed.